app/views/bots/new.html.erb

```diff
@@ -1,91 +1,80 @@
-<% if current_developer && (current_developer.bots.count >= 1 && !current_developer.verified) %>
+<% if (current_developer && (current_developer.bots.count >= 1 && !current_developer.verified)) || current_guest  %>
```

Comment:
This should be done in the BotsController. There's a before_action that redirects guests already. But it's showing the wrong message for when a guest tries to access that page. I suggest creating a new issue for making this validation in the controller and not in the frontend. What do you think about a flash message for the bots limit instead of a new page?